checkov/secrets/runner.py

```diff
@@ -124,7 +125,12 @@ def run(
         # load runnable plugins
         customer_run_config = bc_integration.customer_run_config_response
         plugins_index = 0
-        work_path = str(os.getenv('WORKDIR', current_dir))
+        work_dir_obj = None
+        work_path = str(os.getenv('WORKDIR')) if os.getenv('WORKDIR') else None
```

Comment:
os.getenv returns None if doesn't exist.. so you can use it
you're changing a behavior of using the current dir, are you sure we want to do it ?

Reply:
yes :) 
Check ISSUE BCE-15441